recipes/extra-cmake-modules/all/conandata.yml

```diff
@@ -0,0 +1,19 @@
+sources:
+  "5.75.0":
```

Comment:
Do you need all those versions personally?

Reply:
no, latest is sufficient but I thought they may be useful for someone else.

Reply:
Let's only add the newest version until someone actually request them